scripts/install-connectedk8s.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+CONNECTEDK8S_VERSION="${CONNECTEDK8S_VERSION:-0.3.5}"
+
+az account set --subscription="$SUBSCRIPTION" > /dev/null 2>&1
```

Comment:
If you don't want to use `az account set ...` you can pass `--subscription=...` to all the other `az` CLI commands - just FYI